lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc

```diff
@@ -415,8 +415,13 @@ function paraneue_dosomething_preprocess_node_campaign(&$vars) {
     // Add $campaign_scholarship variable.
     paraneue_dosomething_preprocess_node_campaign_scholarship($vars);
 
+    // Construct the proper end date text
+    $end_date = new DateTime($vars['field_high_season'][0]['value2']);
+    $vars['campaign_end_date'] = $end_date->format('F') . ' ' . $end_date->format('j') . $end_date->format('S');
+
```

Comment:
For reference this is based on the logic already being used in dosomething_campaign.helpers `dosomething_campaign_get_hot_time_left()`